app/templates/application.hbs

```diff
@@ -1,7 +1,5 @@
 {{page-title "Entity Resolution PoC | Publiq"}}
 
-{{outlet}}
-
 <BsNavbar @type="dark" @backgroundColor="primary" as |navbar|>
```

Comment:
Probably best to hide the entire nav bar for a non-authenticated user since nothing will be reachable.

Reply:
The nav bar entries have been hidden, The "outer shell" of the navbar has been kept. This way the layout stays consistent throughout, with the name of the application shown at all times.